### PR TITLE
Periodically refresh voters list

### PR DESCRIPTION
**Pull request by andrewda**
This will refresh the voters list every 30 seconds to keep it up to date. In addition, I started using /voters.json again until we can figure out what's wrong with the /voters page.

(revote of #510)

### CODE REVIEW

**Review by mdcfe — APPROVED** [review]

**Review by PlasmaPower — APPROVED** [review]

**Comment by andrewda** [comment]
Needs another meritocracy approval @PlasmaPower @md678685

**Review by PlasmaPower — APPROVED** [review]

**Comment by PlasmaPower** [comment]
The meritocracy mention system is working, so that isn't necessary anymore.

File: server/static/js/chaos.js [hunk 1]
```diff
@@ -140,40 +140,46 @@ document.getElementById("main").onclick = () => {
...
+                const list = [];
+                for (let j = 0; j < keys.length; j += 1) {
+                    list.push({
+                        names: keys[j],
+                        votes: values[j],
+                    });
+                }
+
+                // eslint-disable-next-line no-nested-ternary
+                list.sort((a, b) => -((a.votes < b.votes) ? -1 : ((a.votes === b.votes) ? 0 : 1)));
```
Comment (PlasmaPower) [inline server/static/js/chaos.js]: You can replace his with `list.sort((a, b) => b.votes - a.votes);`

**Comment by PlasmaPower** [comment]
Should "neutral" comment reviews override an approval? Currently they do.

**Comment by andrewda** [comment]
Probably not.

File: server/static/js/chaos.js [hunk 2]
```diff
@@ -144,8 +144,7 @@ function getText() {
     const result = document.getElementById("result");
     // read text from URL location
     const request = new XMLHttpRequest();
-    // request.open("GET", "voters?amount=20", true);
-    request.open("GET", "voters.json", true);
+    request.open("GET", "api/voters?amount=20", true);
```
Comment (PlasmaPower) [inline server/static/js/chaos.js]: Isn't this still broken?

Comment (andrewda) [reply server/static/js/chaos.js]: Supposedly fixed in #515, but doesn't look like it. So I'll revert back to the `json`s.

**Review by PlasmaPower — APPROVED** [review]

**Pull request merged**